roles/reproducer/tasks/push_code.yml

```diff
@@ -123,6 +126,26 @@
         loop_var: repository
         label: "{{ repository.src | basename }}"
 
+- name: Push code in ~/src to the controller-0
```

Comment:
If someone uses `cifmw_reproducer_skip_fetch_repositories` this `~/src` may not be true anymore.

Reply:
good catch, I missed that